tests/Makefile

```diff
@@ -0,0 +1,93 @@
+# Constants
+MAKEFILE_DIR:=$(dir $(realpath $(firstword ${MAKEFILE_LIST})))
+REPO_ROOT:=$(realpath ${MAKEFILE_DIR}../)
+
+# Initcontainer language under test
+INITCONTAINER_LANGUAGE:=${INITCONTAINER_LANGUAGE}
+
+# Path to the local copy of the repo newrelic/k8s-agents-operator.
+# Only required if building and testing the operator from source.
+LOCAL_OPERATOR_REPO_PATH:=${REPO_ROOT}/../k8s-agents-operator
+
+.PHONY: default
+default: test
+
+# Start minikube
+.PHONY: minikube
+minikube:
+	minikube start --driver=docker
+
+# Install jetstack/cert-manager dependency
+.PHONY: cert-manager
+cert-manager:
+	helm repo add jetstack https://charts.jetstack.io --force-update
+	helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.14.5 --set installCRDs=true
+	kubectl wait --for=condition=Ready -n cert-manager --all pods
+
+# Install official k8s-agents-operator repo
+.PHONY: operator
+operator: cert-manager
+	helm uninstall k8s-agents-operator --ignore-not-found
+	helm repo add k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
+	helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
+		--namespace=default \
+		--set=licenseKey=${NEW_RELIC_LICENSE_KEY} \
```

Comment:
Potential improvement if you want to avoid having to either:
1. pass `NEW_RELIC_LICENSE_KEY` to the make invocation each time you call a target that requires it
2. export `NEW_RELIC_LICENSE_KEY` to your local environment

Is to create a `secrets.mk` file and put the key there, then include that secret file at the top of this makefile.

For example: 

`-include secrets.mk` (the `-` at the beginning allows `make` to continue if the file isn't present)

where in that file you have

`NEW_RELIC_LICENSE_KEY := <INSERT LICENSE KEY HERE>`

along with any other secrets we may end up needing to pass. Then just make sure to add `secrets.mk` to the `.gitignore` and you've saved the headache of ever having to remember to pass/export secrets again.

Reply:
I don't particularly follow how to do that, but would you like to do a follow up PR and add that for your team's use?

We all have the license key exported at all times for our normal work so this hadn't come up.

You could probably do something with `ifeq` like I did for the check at the bottom that loads the secret from a file? Or just do an `include .env` and put any secrets in a .env file, since Makefiles understand that.

Reply:
Oh after reading your comment I realized the `.mk` file is the same as a `.env` file with a different name. I don't particularly care so feel free to add whatever line you need in to make that work in a new PR and I can approve it.